test/unit/events.spec.ts

```diff
@@ -135,6 +137,65 @@ describe("Events", () => {
     });
   });
 
+  describe("events can be stopped almost immediately", () => {
+    let timer: SinonFakeTimers;
+    beforeEach(() => {
+      timer = sinon.useFakeTimers();
+      flickingInfo = createFlicking(horizontal.full);
+    });
+    afterEach(() => {
+      timer.restore();
+    })
+
+    it("holdStart can be stopped almost immediately", () => {
+      // Given
+      let holdCount = 0;
+      const flicking = flickingInfo.instance;
+      flicking.on(EVENTS.HOLD_START, e => {
+        expect(flicking.isPlaying()).to.be.false;
+        holdCount += 1;
+        e.stop();
+      });
+
+      // When
+      simulate(flickingInfo.element, { deltaX: -70, duration: 30 });
+      timer.tick(50);
+
+      simulate(flickingInfo.element, { deltaX: -70, duration: 30 });
+      timer.tick(500);
```

Comment:
@WoodNeck Why tick count is different from above(50)?

Reply:
Previously, it was impossible to trigger holdStart before virtual animating ends if holdStart is stopped.
To test it is resolved, I made test to trigger holdStart twice in a really short term between.
So test can be success only if animation ends between first `tick()`
Second one is just to make sure animation ends after second holdStart.
Actually, it's useless I think.

Reply:
@WoodNeck It would be better to comment it like `make sure animation ends after second holdStart`.

Reply:
That is,
it's not like something directly related to this commit but it's more like testing Axes is working correctly, which is like checking whether using correct version of Axes or so.

Reply:
That would be nice, I'll fix it right away.